packages/ui/react-ui-list/src/components/Tree/TreeItem.tsx

```diff
@@ -196,7 +199,7 @@ export const RawTreeItem = <T = any,>({
           isBranch && open && handleOpenChange();
           break;
         case ' ':
-          handleSelect();
+          handleSelect(event.altKey);
```

Comment:
Did you choose alt for a particular reason? I might have tried shift…

Reply:
mostly because "alternate" behaviour/secondary "option", would be happy with shift as well if folks think that makes more sense

Reply:
I don’t necessarily have an opinion, would defer to user testing

Reply:
While using mouse and keyboard, I guess I’d want to change the cursor to indicate what it will do – maybe leave a todo about that?

Reply:
Maybe this is more of a global feature worth a separate issue

Reply:
add hint to tooltip?

Reply:
> add hint to tooltip?

This is the best solution so far, yes ✨